src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,64 @@
+package lotto.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import lotto.domain.generator.LottoAutoGenerator;
+import lotto.domain.lotto.Answer;
+import lotto.domain.lotto.PrizeRatio;
+import lotto.domain.lotto.Rank;
+import lotto.domain.lotto.Ticket;
+import lotto.domain.lotto.Tickets;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class LottoController {
+
+    private static final int PRICE_PER_ONE_TICKET = 1000;
+
+    private static LottoController lottoController = null;
+
+    private LottoController() {}
+
+    public static LottoController getInstance() {
+        if (lottoController == null) {
+            lottoController = new LottoController();
+        }
+        return lottoController;
+    }
+
+    public void run() {
+        Tickets tickets;
+        int amount = InputView.getAmount();
+        int count = amount / PRICE_PER_ONE_TICKET;
+
+        tickets = purchase(count);
```

Comment:
`Tickets`객체 선언을 여기서해주는 건 어떨까요?

```suggestion
        int amount = InputView.getAmount();
        int count = amount / PRICE_PER_ONE_TICKET;

        Tickets tickets = purchase(count);
```